build/ci-build.yml

```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
 trigger:
   branches:
     include:
-    - master
+      - master
```

Comment:
Is this on purpose? I don't think we need this, do we? 🤔
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example#triggers

Reply:
Ah, no; I don't think so. We had this before on all CI builds; but as it turns out, this is triggered by default.

Reply:
I think we actually need them but I was more referring to the additional spaces